README.md

```diff
@@ -25,6 +25,7 @@ git submodule foreach git pull origin master
      * [Kubernetes](#kubernetes)
      * [User Installation](#user-installation)
      * [Root Installation](#root-installation)
+     * [Script Installation](#script-installation)
```

Comment:
Not sure about the section name, “Script installation” is kind of misleading,  we're not really installing scripts but fluentd + plugins. @Turhsus @MahithaB  any ideas on how to name this section.

Reply:
Why not just that? (#fluent-plugin-installation)